cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -2829,6 +2829,14 @@
     </description>
   </property>
 
+  <property>
+    <name>metrics.processor.offer.timeout.ms</name>
```

Comment:
Do we need a precision of milliseconds or would seconds suffice?

Reply:
Time intervals in MS are a good practice. You never know when you'll need sub-second granularity

Reply:
I'd give an option to make it subsecond (e.g. 0.5 seconds), at least for tests. Also other related setting (metrics.processor.max.delay.ms) is in ms